pkg/windows/windows.go

```diff
@@ -474,7 +480,13 @@ func (vm *windows) Configure() error {
 		return errors.Wrapf(err, "error configuring containerd")
 	}
 
-	return vm.runBootstrapper()
+	if err := vm.ensureWICDSecretContent(serviceAccountCA, serviceAccountToken); err != nil {
+		return err
+	}
+	wicdBootstrapCmd := fmt.Sprintf("%s bootstrap --desired-version %s --api-server %s --sa-ca %s%s --sa-token %s%s",
+		wicdPath, desiredVer, apiServerURL, K8sDir, wicdCAFile, K8sDir, wicdTokenFile)
+	_, err := vm.Run(wicdBootstrapCmd, true)
```

Comment:
Why is the output being ignored? Shouldn't we display it as a debug? Won't it be useful to include it in the error message?